addons/pos_sale/static/tests/tours/pos_sale_tours.js

```diff
@@ -23,4 +23,20 @@ odoo.define('pos_sale.tour', function (require) {
     Chrome.do.clickTicketButton();
 
     Tour.register('PosSettleOrder', { test: true, url: '/pos/ui' }, getSteps());
+
+    startSteps();
+
+    ProductScreen.do.confirmOpeningPopup();
+    ProductScreen.do.clickQuotationButton();
+    ProductScreen.do.selectFirstOrder();
+    ProductScreen.do.clickOrderline("Product B", 1);
+    ProductScreen.do.pressNumpad('Qty 0');
+    ProductScreen.do.clickOrderline("Product A", 1);
+    ProductScreen.do.pressNumpad('Qty 1');
```

Comment:
Instead of setting the quantity to 1 which I believe is already 1, maybe it's better to check whether the quantity of orderline Product A is 1.

```
    ProductScreen.check.selectedOrderlineHas('Product A', '1');
```